lib/exec.js

```diff
@@ -149,9 +149,23 @@ export function executeDirect(command, args, options, filenameTransform) {
     });
 }
 
+export function getNsJailCfgFilePath(configName) {
+    const configNamingMap = {
+        sandbox: {key: 'nsjailConfigSandbox', default: 'etc/nsjail/sandbox.cfg'},
```

Comment:
nit: I'd rather either not specify a default here (and make it a fatal error if `execProps` returns undefined or whatever), or remove the defaults from the properties files - as it stands now we would have to change the defaults in two places.

Slight preference for leaving the values in the properties files since it improved discoverability for users.